tests/microos/networkmanager.pm

```diff
@@ -151,6 +154,8 @@ sub test_flags {
 }
 
 sub post_fail_hook {
+    script_run('journalctl -u NetworkManager -b >> /var/log/nmcli_logs');
```

Comment:
`-b --boot[=ID]` you are miss the ID of the current boot

Reply:
Thanks, I just copied the line